config/crossSections13p6TeV.yaml

```diff
@@ -0,0 +1,718 @@
+  crossSec: 0.001111
+  reference: XSDB DYto2Tau_MLL-1500to2500_TuneCP5_13p6TeV_powheg-pythia8 modifiedOn 2024-04-03 19:20:34
+  unc: 1.787e-07
+DYto2Tau_MLL-2500to4000_powheg:
+  crossSec: 5.949e-05
+  reference: XSDB DYto2Tau_MLL-2500to4000_TuneCP5_13p6TeV_powheg-pythia8 modifiedOn 2024-04-03 19:20:38
+  unc: 9.162e-09
+DYto2Tau_MLL-4000to6000_powheg:
+  crossSec: 1.558e-06
+  reference: XSDB DYto2Tau_MLL-4000to6000_TuneCP5_13p6TeV_powheg-pythia8 modifiedOn 2024-04-03 19:20:41
+  unc: 2.078e-10
+DYto2Tau_MLL-6000_powheg:
+  crossSec: 3.519e-08
+  reference: XSDB DYto2Tau_MLL-6000_TuneCP5_13p6TeV_powheg-pythia8 modifiedOn 2024-04-03 19:20:47
+  unc: 6.811e-12
+
+DYto2TautoMuTauh_M-50_madgraphMLM:
+  crossSec: 53.28
+  reference: XSDB DYto2TautoMuTauh_M-50_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2023-03-26 13:56:19
+  unc: 0.5212
+
+GluGluHToTauTau_M125:
+  crossSec: 52.23 * (0.06272)
+  reference: https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCHWG136TeVxsec_extrap
+  unc: +4.6% -6.7% (theory) +-3.9 (TH Gaussian) +-3.2 (PDF+alpha s) +-1.9 (PDF) +-2.6 (alpha s)
+GluGluHToTauTau_M125_amcatnloFXFX:
+  crossSec: 52.23 * (0.06272)
+  reference: https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCHWG136TeVxsec_extrap
+  unc: +4.6% -6.7% (theory) +-3.9 (TH Gaussian) +-3.2 (PDF+alpha s) +-1.9 (PDF) +-2.6 (alpha s)
+GluGluHToWWTo2L2Nu_M125:
+  crossSec: 52.23 * (0.2137) * (3 * 0.1086)
+  reference: https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCHWG136TeVxsec_extrap
+  unc: +4.6% -6.7% (theory) +-3.9 (TH Gaussian) +-3.2 (PDF+alpha s) +-1.9 (PDF) +-2.6 (alpha s)
+
+
+QCD_HT40to70_MLM:
+  crossSec: 311400000.0
+  reference: XSDB QCD-4Jets_HT-40to70_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 19:46:52
+  unc: 843600.0
+QCD_HT70to100_MLM:
+  crossSec: 58500000.0
+  reference: XSDB QCD-4Jets_HT-70to100_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 19:46:55
+  unc: 167400.0
+QCD_HT100to200_MLM:
+  crossSec: '0.0'
+  reference: fill me!
+  unc: ''
+QCD_HT200to400_MLM:
+  crossSec: 1961000.0
+  reference: XSDB QCD-4Jets_HT-200to400_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 19:46:48
+  unc: 5862.0
+QCD_HT400to600_MLM:
+  crossSec: 95620.0
+  reference: XSDB QCD-4Jets_HT-400to600_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 19:46:50
+  unc: 289.3
+QCD_HT600to800_MLM:
+  crossSec: 13540.0
+  reference: XSDB QCD-4Jets_HT-600to800_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 19:46:54
+  unc: 41.11
+QCD_HT800to1000_MLM:
+  crossSec: 3033.0
+  reference: XSDB QCD-4Jets_HT-800to1000_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 19:46:57
+  unc: 9.206
+QCD_HT1000to1200_MLM:
+  crossSec: 883.7
+  reference: XSDB QCD-4Jets_HT-1000to1200_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 19:46:40
+  unc: 2.69
+QCD_HT1000To1200_MLM:
+  crossSec: 883.7
+  reference: XSDB QCD-4Jets_HT-1000to1200_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 19:46:40
+  unc: 2.69
```

Comment:
```suggestion
QCD_HT1000to1200_MLM:
  crossSec: 883.7
  reference: XSDB QCD-4Jets_HT-1000to1200_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 19:46:40
  unc: 2.69
```

Reply:
Fixed, but I really don't understand how this one happened (maybe some copy paste by hand mistake?)